src/blocks/content-helper/performance-details/component.tsx

```diff
@@ -43,7 +43,7 @@ function PerformanceDetails() {
 					setLoading( false );
 				} )
 				.catch( async ( err ) => {
-					if ( retries > 0 ) {
+					if ( retries > 0 && err.retryFetch ) {
```

Comment:
Here the expectation is to retry or not if the error is not in `noRetryList`?

As of now it seems like we cannot retry in this case.

Reply:
This works as expected in my testing. Note that we're doing `this.retryFetch = ! noRetryFetchErrors.includes( this.code );`. So if the error is not in the (exclusion) array, we are retrying it.

Reply:
Okay. Got it ... we are retrying only once now.